docs/guides/ipsec-guide.md

```diff
@@ -0,0 +1,129 @@
+# IPsec Offload Guide
+
+## 1. Overview
+
+The [ipdk-io/ipsec-recipe](https://github.com/ipdk-io/ipsec-recipe) is an
+independent IPDK recipe, which provides strongSwan plugin code to configure and
+program control messages into the target.
+Consult the ipsec-recipe for full setup information.
+
+This document provides information pertaining to the gRPC server and the
```

Comment:
I'd delete this paragraph (sentence). It's introductory boilerplate. Use the space for something more substantial.